bin/next-init

```diff
@@ -12,14 +12,18 @@ const argv = parseArgs(process.argv.slice(2), {
 
 const dir = resolve(argv._[0] || '.')
 
-exists(join(dir, 'package.json'))
+exists(dir)
 .then(async present => {
+  if (!present) {
+    await mkdir(dir)
```

Comment:
Ahh, yep that's a good point.
Afaict `-p` is not provided out of the box by `mz/fs` so I'll have to pull in a lib